api/cluster-resources/playground-extension.js

```diff
@@ -0,0 +1,43 @@
+// Workbench "Playground Mode" extension. In "Playground Mode", changes are not
+// saved back to GCS. This extension makes minor UI tweaks to differentiate this
+// mode from normal Workbench Jupyter usage. For now, this also removes/hides
+// save controls. Technically one can still save changes locally (and changes
+// are automatically saved periodically), but as a first pass we just want to
+// avoid having to explain this nuance.
+
+define([
+    'base/js/namespace'
+], (Jupyter) => {
+  const load = () => {
+    // This prefix must be kept in sync with the Workbench localization API,
+    // see https://github.com/all-of-us/workbench/blob/master/api/src/main/java/org/pmiops/workbench/api/ClusterController.java
+    const nbPath = Jupyter.notebook.notebook_path;
+    if (!nbPath.startsWith('workspaces:playground/')) {
+      return;
+    }
+
+    // Disable UI controls/notifications relating to saving.
+
+    // "notbook" is an intentional typo to match Jupyter UI HTML.
```

Comment:
This isn't a comment on your PR but... really? Wow.

Reply:
Yep. They'd probably break people at this point if they changed, given how extensive some of the UI extensions probably are.